spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/ConsumerProperties.java

```diff
@@ -125,5 +127,13 @@ public HeaderMode getHeaderMode() {
 	public void setHeaderMode(HeaderMode headerMode) {
 		this.headerMode = headerMode;
 	}
+
+	public boolean isSupportDeserializationNatively() {
```

Comment:
> there is no way of bypassing the codec at the consumer side I think. 

I don't quite understand. If the native decoder/deserializer kicks in, the message payload won't be a `byte[]` (normally) - so the codec shouldn't handle it (it can't handle anything that's not a `byte[]`).